lib/Tools/LinearLayout.cpp

```diff
@@ -958,8 +958,6 @@ LinearLayout LinearLayout::invertAndCompose(const LinearLayout &outer) const {
 
 LinearLayout LinearLayout::invert() const {
   // A^-1(x) = A^-1(I(x)), thus A.invert() = I.invertAndCompose(A)
-  assert(isInvertible() &&
```

Comment:
@Jokeren @lezcano please let me know if you'd rather I create a `LinearLayout::pseudoinvert` function instead

Reply:
Can you create `pseudoinvert`? I think it's nice to have an API that's slightly stricter (even if functionally equivalent) to catch errors.

Reply:
Will do!